### ISSUE TITLE
Language server is broken? 

### ISSUE THREAD

**Issue by shcheklein** [iterative/vscode-dvc#2459]
<img width="447" alt="Screen Shot 2022-09-24 at 5 08 22 PM" src="https://user-images.githubusercontent.com/3659196/192122902-7e6ced8e-b957-40fe-be9e-4688ea0bc4da.png">

VS Code version:

```
Version: 1.71.2 (Universal)
Commit: 74b1f979648cc44d385a2286793c226e611f59e7
Date: 2022-09-14T21:05:37.721Z (1 wk ago)
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: No
```

Extension version: v0.4.6

Output:

```
[Info  - 5:08:14 PM] Connection to server got closed. Server will restart.
[Error - 5:08:14 PM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 5:08:14 PM] DVC Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 5:08:14 PM] Restarting server failed
  Message: Pending response rejected since connection got disposed
  Code: -32097 
node:internal/modules/cjs/loader:990
  throw err;
  ^

Error: Cannot find module 'vscode-languageserver/node'
Require stack:
- /Users/ivan/.vscode/extensions/iterative.dvc-0.4.6/dist/node_modules/dvc-vscode-lsp/dist/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at Module._load (node:internal/modules/cjs/loader:832:27)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/ivan/.vscode/extensions/iterative.dvc-0.4.6/dist/node_modules/dvc-vscode-lsp/dist/server.js:1:82)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ivan/.vscode/extensions/iterative.dvc-0.4.6/dist/node_modules/dvc-vscode-lsp/dist/server.js'
  ]
}
[Error - 5:08:14 PM] The DVC Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
[Error - 5:08:14 PM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 5:08:14 PM] DVC Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 5:08:14 PM] Restarting server failed
  Message: Pending response rejected since connection got disposed
  Code: -32097 
```

**Comment by mattseddon** [comment]
My first guess would be this is happening because the language server is not being copied into the extension before it is bundled with `vsce`. I will raise a PR to stop the server from starting. We can ship that and it will patch the problem until it is fixed.

**Comment by mattseddon** [comment]
@wolmir if my suspicions are correct the best thing to do would be to add Webpack to the language server and copy the way that the webviews are packaged with the extension.

**Comment by wolmir** [comment]
@mattseddon @shcheklein 
I'll add Webpack to the server module this morning, following the configurations we already have

**Comment by mattseddon** [comment]
Fixed by #2469
